env/env_basic_test.cc

```diff
@@ -374,6 +327,32 @@ TEST_P(EnvMoreTestWithParam, GetChildren) {
   ASSERT_EQ(0U, children.size());
 }
 
+TEST_P(EnvMoreTestWithParam, GetChildrenIgnoresDotAndDotDot) {
+  auto* env = Env::Default();
+  ASSERT_OK(env->CreateDirIfMissing(test_dir_));
```

Comment:
Are you guaranteed the test_dir is empty when you enter this method, or do you need to check that the number of children has increased by 1?

Reply:
Yes, I think that `EnvBasicTestWithParam::TearDown` takes care of that.